### PR TITLE
add support for apps in subdirs and make sure to copy static resources in build

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

File: lib/src/barback/dartdevc/dartdevc_environment.dart [hunk 1]
```diff
@@ -83,6 +87,15 @@ class DartDevcEnvironment {
       allFutureAssets
           .addAll(_buildAsset(module.jsId, logError: logError).values);
     }
+    // Copy all js resoureces for each of the app dirs that were discovered.
```
Comment (munificent) [inline lib/src/barback/dartdevc/dartdevc_environment.dart]: "JS". 😛

File: lib/src/barback/dartdevc/dartdevc_environment.dart [hunk 2]
```diff
@@ -83,6 +87,15 @@ class DartDevcEnvironment {
...
+    for (var dir in appDirs) {
+      allFutureAssets.addAll([
+        _buildJsResource(new AssetId(_packageGraph.entrypoint.root.name,
+            p.url.join(dir, 'dart_sdk.js'))),
+        _buildJsResource(new AssetId(
+            _packageGraph.entrypoint.root.name, p.url.join(dir, 'require.js'))),
+      ]);
```
Comment (munificent) [inline lib/src/barback/dartdevc/dartdevc_environment.dart]: Nit: This wouldn't be any longer as two calls to `.add()`.

File: lib/src/barback/dartdevc/dartdevc_environment.dart (hunk 1 above)
Comment (natebosch) [inline lib/src/barback/dartdevc/dartdevc_environment.dart]: At this point you might want to write a github-pr bot that adds this comment for you

Comment (jakemac53) [inline lib/src/barback/dartdevc/dartdevc_environment.dart]: lol, done

File: lib/src/barback/dartdevc/dartdevc_environment.dart (hunk 2 above)
Comment (jakemac53) [inline lib/src/barback/dartdevc/dartdevc_environment.dart]: done, and I used cascades just for you :D

**Pull request merged**